modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v1/services/Boxes.scala

```diff
@@ -379,6 +391,28 @@ object Boxes {
         .thrushK(trans.xa)
     }
 
+    def getAllUnspentOutputs(address: Address, paging: Paging, ord: SortOrder): F[Items[UOutputInfo]] = {
+      val ergoTree = addressToErgoTreeHex(address)
+      (for {
+        nConfirmed  <- outputs.countUnspentByErgoTree(ergoTree)
+        nUnonfirmed <- uoutputs.sumUnspentByErgoTree(ergoTree)
+        confirmed <- outputs
+                       .streamUnspentByErgoTree(ergoTree, paging.offset, paging.limit, ord.value)
+                       .chunkN(serviceSettings.chunkSize)
+                       .through(toOutputInfo)
+                       .map(UOutputInfo.fromOutputInfo)
+                       .to[List]
+        unconfirmed <- Stream
+                         .evalSeq(
+                           uoutputs
+                             .getAllUnspentByErgoTree(ergoTree)
+                         )
```

Comment:
So it seems your paging doesn't work as it should here. You retrieve a slice of expected number of confirmed outputs `streamUnspentByErgoTree(ergoTree, paging.offset, paging.limit, ord.value)` and then concatenate it with unknown number of unconfirmed outputs.